### DISCUSSION TITLE
Losing site of the bigger picture in Sunflower Land

### DISCUSSION THREAD

**Discussion by FarmerKuenster** [sunflower-land/sunflower-land#2429] [General]
I write this from the perspective of someone who has been with the game since the early days, and  I've always tried kept up what was being posted in discord, and here in Github. I came into this game not knowing really anything about crypto or how a game like this would work. All I knew was that I loved farming, the concept seemed cool and I wanted to be involved.

**The problems I see as a player**

- The focus of the game seems to be so honed in on the statistical side right now. (lots of discussions about boost info for in game NFT's, node balancing, sync, and burning of resources and how to max out your perks on your land)

- A big focus on "fairness" node balancing, more and more NFT's and getting better perks. (personally I didn't care if it was "fair or not", I saw it as a future opportunity to maybe trade land/nodes with people which could have been really cool)

- catering to pushing out features/content for players who really only see SFL as a way to make a quick buck. (This is dangerous because then the devs are no longer developing the game for their actual target audience.)

- It feels like a game loop of get this new "thing/perk" to help you get more resources to then save up and buy the next "thing/perk" then repeat that over and over. (feels redundant and makes me say whats the point?)

- lack of lore/storytelling in the game. ie. why are there different islands, more lore between the humans and goblins. who is asking me to do a quest, why should I help them? 

-Dedicated Players leaving the game because of the direction SFL has taken with development

For us players who are invested in the game, not from a monetary perspective, but emotionally, who see the potential of this game in the crypto space, I ask the question:    **What is the bigger picture?**

For the game to succeed there needs to be bigger goals for us players to work towards, lore to draw us in, WHY should we help the character with said quest. A way to burn resources in game that Isn't directly linked to just leveling up and buying more NFT's. The game won't be able to sustain itself on the current linear progression. 

Whatever needs to happen to keep the game alive and thriving should be perused by the devs. If hiring someone who specializes in Lore/storytelling, hire them. If the game needs a legit video game designer to come and add their expertise, hire them. I'm a firm believer that if you need a skill that Isn't in your wheelhouse find someone who can help.

@adamhannigan Me and so many of the original players want to see Sunflower Land be a huge success in the Crypto space. Whatever your initial vision was/is for the game, please don't lose site of it.

**Comment by future2back** [comment]
I believe that this type of feedback is highly valuable for the game's future. The game lacks entertainment is visible and there is a lack of improving this aspect.

**Reply by FarmerKuenster** [reply]
I just hope that they see what the future of the game will be if this is the road they continue down.

**Comment by TheDcol** [comment]
> catering to pushing out features/content for players who really only see SFL as a way to make a quick buck. (This is dangerous because then the devs are no longer developing the game for their actual target audience.)

> It feels like a game loop of get this new "thing/perk" to help you get more resources to then save up and buy the next "thing/perk" then repeat that over and over. (feels redundant and makes me say whats the point?)

Excellent points and I totally agree with. I would be all for the game to throw me some challenges that are not just about minting new nfts or participating in seasonal events (to get more nfts..)

I want to face decisions within the game, I want to use my crops/eggs for stuff that won't directly profit me. Be it random events or a farm maintenance system.

**Reply by FarmerKuenster** [reply]
Yes! "_I want to use my crops/eggs for stuff that won't directly profit me._" great way to word it! Not everything needs to be focused on profit. Decisions that make the game no longer linear would be amazing. Id love to talk to other farmers/bumpkins and feel like everyone's farming journey is unique.

**Reply by Thebyber** [reply]
An attractive mechanic could be the experience in each skill branch to acquire some building or decoration.

Now that landscalping comes... we could unlock the wooden fences from the helios store if we have cut down 100 trees, if by chance we have cut down 1000 trees we can unlock another wooden fence with a more exclusive design.

The idea is to incentivize the game with an achievement that provides us with the improvement of a building (for example, to improve the toolshed we would need to cut down 1000 trees and collect 200 stones; giving a benefit at level 2 of 75% of tools instead of the current 50%) or allow us to buy in helios some exclusive decoration in terms of our effort on the farm.

I used the translator since my level of English is not very good... I'm very sorry!

**Comment by rasheed19871** [comment]
i agree

**Comment by adamhannigan** [comment]
Thanks for raising your concerns @FarmerKuenster 

This is something the team has been discussing and planning around the last 4-5 months. We've been doing some big upgrades that means there has been less focus on content (narrative devices, community events + cyclical features).

We are always going to release features (multiple bumpkins, magic beans, fishing etc), so I don't include this as the 'vision'.

Overall where we are heading is towards:
- Seasons with emotional engagement + storytelling to build connections. We can't spoil too much, but Dawn Breaker is going to be completely different than Solar Flare. It will be an actual gameplay adventure.
- Social farming + collaboration 
- User generated content. Create your own islands, art, mechanics + art. We've got a tremendous player base of people with ideas, art skills and development skills.

Beyond that we have got lots of incredible ideas. Being in this industry roadmaps change every 6 months to stay up to date with the latest developments - we can't promise anything, but we're listening a lot to the players and hope you will enjoy the decisions we make and content we deliver :)

**Comment by Chucrutty** [comment]
What most caught my attention in the game when I joined was the promise that you could expand infinitely and build your empire, but what I'm seeing is a huge problem of burning resources, a highly harmful player impoundment for the game, the game It can't just be accumulating resources... it doesn't make sense...
![image](https://user-images.githubusercontent.com/125780525/232936821-cd0e5cde-b058-47d1-849b-f6fc4564174b.png)
i didnt buy any thing, and have 1MM XP on food, frustrating and disheartening to feel stuck in the game, I'm still in the game, because I really believe in the team, but I also think they're losing time to launch things, which can be very risky because the games market and even more so Crypto needs enormous agility, I'm very sad about the game in the last 3 months

**Comment by Chucrutty** [comment]
![image](https://user-images.githubusercontent.com/125780525/232940597-6b6eada2-043f-4b66-b1d7-3a75b6a76874.png)

I wish I never see this message again

**Reply by wersout** [reply]
You do understand econamy balancing is important for the game, right??
Releasing free expansion how ever you want would inflate everything with the player base we have so the focus devs give to 
on-ramp is what's needed for the game not you to expand how ever you like so you need to keep your patiens.......
We get what we pay for, everyone want to pay as little as possible in sync fees and that lead to the team stay small and everything takes longer so we have to be patient for everything... 
Set sync fee to 1-2 dollars or a monthly cost of 20$ so they can hire more developer or as they do now focusing on on-ramp to get more players in to the game
